code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -20,6 +20,8 @@
 	var/blood_mask = 'icons/mob/human_races/masks/blood_human.dmi'
 
 	var/prone_icon                                       // If set, draws this from icobase when mob is prone.
+	var/icon_x_offset
```

Comment:
To avoid issues with typing, these should be initialized as `var/icon_x_offset = 0`. Null is not the same as 0.